core/client/fs/src/test/java/alluxio/client/file/BaseFileSystemTest.java

```diff
@@ -567,8 +558,49 @@ public void uriCheckNoSchemeAuthority() throws Exception {
     AlluxioURI uri = new AlluxioURI("/root");
     mFileSystem.createDirectory(uri);
 
-    assertFalse(mTestLogger.wasLogged("The URI authority"));
-    assertFalse(mTestLogger.wasLogged("The URI scheme"));
+    assertFalse(loggedAuthorityWarning());
+    assertFalse(loggedSchemeWarning());
+  }
+
+  @Test
+  public void uriCheckZkAuthorityMatch() throws Exception {
+    configureZk("a:0,b:0,c:0");
+    useUriWithAuthority("zk@a:0,b:0,c:0"); // Same authority
+    useUriWithAuthority("zk@a:0;b:0+c:0"); // Same authority, but different delimiters
+  }
+
+  @Test
+  public void uriCheckZkAuthorityMismatch() throws Exception {
+    configureZk("a:0,b:0,c:0");
+
+    assertBadAuthority("a:0,b:0,c:0", "Should fail on non-zk authority");
+    assertBadAuthority("zk@a:0", "Should fail on zk authority with different addresses");
+    assertBadAuthority("zk@a:0,b:0,c:1", "Should fail on zk authority with different addresses");
+  }
+
+  private void assertBadAuthority(String authority, String failureMessage) throws Exception {
+    try {
+      useUriWithAuthority("zk@a:0,b:0,c:1");
```

Comment:
should the `zk@a:0,b:0,c:1` be change to `authority`?

Reply:
oh no my test is garbage! x.x

thanks for catching this, will push a fix